src/timeUtils.test.ts

```diff
@@ -38,30 +38,32 @@ test('convertDurationStringToUs', () => {
     `);
 });
 
-test('convertUTCToMs', () => {
+test('convertutcToMs', () => {
```

Comment:
Could this be `convertUtcToMs`?

Reply:
yes i can :)

Reply:
Actually that is a typo it is now `utcToMs`